src/main/java/me/gnat008/perworldinventory/commands/SetWorldDefaultCommand.java

```diff
@@ -15,7 +15,7 @@
 public class SetWorldDefaultCommand implements ExecutableCommand {
 
     @Inject
-    private FileWriter fileSerializer;
+    private FlatFile fileSerializer;
```

Comment:
That's actually a really good idea, I hadn't thought of that. I had so far just figured "oh it handles files, put all the methods in there!", but you're right. Especially since they need to be used as well for the setWorldDefaults command.